### PR TITLE
feat:alert-p0 component

### PR DESCRIPTION
**Pull request by nyyshaaa**
alertt

### CODE REVIEW

File: pages/docs.html [hunk 1]
```diff
@@ -34,13 +34,13 @@ <h1 class="nav-h1">Getting Started</h1>
         <h1 class="nav-h1">Components</h1>
         <ul class="components-box">
             <li class="comp-link"><a href="/pages/components/Avatar-badge/avatar-badge.html">Avatar-Badge</a></li>
```
Comment (zabihshaik) [inline pages/docs.html:36]: Better to have Avatar and Badge as separate Components

File: pages/docs.html [hunk 2]
```diff
@@ -34,13 +34,13 @@ <h1 class="nav-h1">Getting Started</h1>
...
-            <li class="comp-link"><a href="#">Button</a></li>
-            <li class="comp-link"><a href="#">Cards</a></li>
-            <li class="comp-link"><a href="#">Image-input</a></li>
-            <li class="comp-link"><a href="#">Chips</a></li>
-            <li class="comp-link"><a href="#">Tabs</a></li>
-            <li class="comp-link"><a href="#">Modal</a></li>
+            <li class="comp-link"><a href="/pages/components/Alert/alert.html">Alert</a></li>
+            <li class="comp-link"><a href="/pages/components/Button/button.html">Button</a></li>
+            <li class="comp-link"><a href="/pages/components/Cards/cards.html">Cards</a></li>
+            <li class="comp-link"><a href="/pages/components/Image-input/image-input.html">Image-Input</a></li>
```
Comment (zabihshaik) [inline pages/docs.html:40]: Better to have Image and Input as separate Components

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a few suggestions:
1. Better to have Avatar and Badge as separate Components (1 Branch is OK for PR, but in the list it is better to be separate)
2. In the Alerts preview, not able to differentiate the alert. Better to have a border or outline.

File: pages/docs.html (hunk 2 above)
Comment (nyyshaaa) [reply pages/docs.html:40]: okay will try to change it.

Comment (nyyshaaa) [reply pages/docs.html:40]: i will add outline for alerts

Comment (zabihshaik) [reply pages/docs.html:40]: Or just use the different background colors. Like primary color, secondary color, etc.

File: pages/components/Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+.alert-display-wrapper{
+    flex-direction: column;
+    padding: 2.5rem;
+  
+}
```
Comment (Rockyraven) [inline pages/components/Alert/alert.css:5]: remove this extra space

**Comment by Rockyraven** [comment]
all goods for me

File: pages/components/Alert/alert.css (hunk 3 above)
Comment (nyyshaaa) [reply pages/components/Alert/alert.css:5]: okay i forgot maybe, will do in end!

**Pull request merged**